src/services/huaweicloud/credential.rs

```diff
@@ -38,3 +38,22 @@ impl Credential {
         true
     }
 }
+
+impl Debug for Credential {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(
```

Comment:
Please use `f.debug_struct` instead (so that we can keep the same style).

Reply:
ok, fixed